eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core SDK'
+    inputs:
+      useGlobalJson: true
```

Comment:
I believe that DevOps has included .NET 5.0 on the box so lets please make sure that this step doesn't continue to download and install it again. We moved away from this step to avoid that networking reliability issues and I'd hate to have that come back.

Reply:
Nope, I've tried and it's not available yet.

Reply:
It is supposed to be coming soon, see https://github.com/actions/virtual-environments/issues/1891. We could consider waiting for that, but if we get this merged in before that work is ready we should file an issue to follow-up and remove this explicit install step once it ends up on the image.

Reply:
#17118